### PR TITLE
Fs 4631 generate form unit tests

### PR DESCRIPTION
**Pull request by srh-sloan**
### Change description
- Unit tests passing for export
- Refactored conditions functions to account for multiple subconditions

- [x] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -12,29 +13,27 @@ services:
...
     environment:
       - POSTGRES_PASSWORD=password
       - POSTGRES_DB=fund_builder
-      - DATABASE_URL=postgresql://postgres:password@fab-db:5432/fund_builder
-      - DATABASE_URL_UNIT_TEST=postgresql://postgres:password@fab-db:5432/fab_unit_test
 
-  form-runner:
-    container_name: form-runner
+  form-runner-dc:
+    container_name: form-runner-dc
```
Comment (nuwan-samarasinghe) [inline docker-compose-dev.yml:36]: bit curious about the dc post fix ?

File: tests/test_form_import.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+from app.shared.data_classes import ConditionValue
+from app.shared.data_classes import SubCondition
+
+
+@pytest.mark.parametrize(
+    "input_condition,exp_result",
+    [
+        (
+            {
+                "displayName": "org type a",
```
Comment (nuwan-samarasinghe) [inline tests/test_form_import.py:22]: Can we add these as a constant from a variable ? to reduce the code lines

**Review by nuwan-samarasinghe — DISMISSED** [review]
LGTM

File: docker-compose-dev.yml (hunk 1 above)
Comment (srh-sloan) [reply docker-compose-dev.yml:36]: dc=docker-compose - I just needed to separate it from the ones in the dev container... (but that was already merged)

**Review by Adam-W1 — APPROVED** [review]
nice work 👍

File: tests/test_form_import.py (hunk 2 above)
Comment (srh-sloan) [reply tests/test_form_import.py]: Thanks - good shout, refactored!

**Review by Adam-W1 — DISMISSED** [review]

**Review by Adam-W1 — DISMISSED** [review]

**Pull request merged**